SU-SWS/stanford_profile
# D8CORE-3263 Create gallery paragraph type with colorbox actions

# READY FOR REVIEW

# Summary
- Gallery paragraph type using colorbox display

# Need Review By (Date)
- 1/14

# Urgency
- medium

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-3263-gallery`
2. `blt drupal:update`
3. create a basic page and add a gallery paragraph type
4. add some images and save
5. verify the images are displayed and if you click on them, they open to a larger image.
   - use https://github.com/SU-SWS/stanford_profile_helper/pull/62 for styling if desired to display as a grid.
# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
